central/networkgraph/flow/datastore/internal/store/postgres/store_test.go

```diff
@@ -95,15 +90,16 @@ func (s *NetworkflowStoreSuite) TestStore() {
 
 	// Adding the same thing twice to ensure that we only retrieve 1 based on serial Flow_Id implementation
 	s.NoError(s.store.UpsertFlows(s.ctx, []*storage.NetworkFlow{networkFlow}, zeroTs))
-	networkFlow.LastSeenTimestamp = getTimestamp(2)
+	networkFlow.LastSeenTimestamp = protocompat.GetProtoTimestampFromSeconds(2)
 	s.NoError(s.store.UpsertFlows(s.ctx, []*storage.NetworkFlow{networkFlow}, zeroTs))
 	foundNetworkFlows, _, err = s.store.GetAllFlows(s.ctx, nil)
 	s.NoError(err)
 	s.Len(foundNetworkFlows, 1)
 	s.Equal(networkFlow, foundNetworkFlows[0])
 
 	// Check the get all flows by since time
-	foundNetworkFlows, _, err = s.store.GetAllFlows(s.ctx, getTimestamp(3))
+	time3 := time.Unix(3, 0)
```

Comment:
Why not `protocompat.GetProtoTimestampFromSeconds(3)` as in the other places where we change from `getTimestamp(#)`?

`time3` is here "3 seconds after January 1, 1970 UTC".